api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
I think this is a fair argument. Given this, can you explain why we're not using `NodeContributorSerializer` (https://github.com/CenterForOpenScience/osf.io/blob/develop/api/nodes/serializers.py#L276) ?